internal/schemautil/userconfig/apiconvert/toapi.go

```diff
@@ -127,6 +127,12 @@ func objectItemToAPI(
 
 	fv := v[0]
 
+	// Object with only "null" fields becomes nil
+	// Which can't be cast into a map
+	if fv == nil {
```

Comment:
we would wanna discuss it internally a bit, since it was made this way on purpose in the current version